core/chains/evm/txmgr/txmgr.go

```diff
@@ -280,7 +280,7 @@ func (b *Txm) HealthReport() map[string]error {
 	return report
 }
 
-func (b *Txm) runLoop(eb *EthBroadcaster, ec *EthConfirmer, enabledAddresses []common.Address) {
```

Comment:
This was just for readability, since we're already inside of a method that has access to these.